ios/Podfile.lock

```diff
@@ -563,6 +563,6 @@ SPEC CHECKSUMS:
   Yoga: 7ab6e3ee4ce47d7b789d1cb520163833e515f452
   YogaKit: f782866e155069a2cca2517aafea43200b01fd5a
 
-PODFILE CHECKSUM: 811937d5c2e3c73f2910e2c19f325c71b5ed0606
+PODFILE CHECKSUM: 156997005009696054d43a562cc1a4db48a126ff
```

Comment:
generally pod files shouldn't be pushed to github but my b because it's not in git ignore. should be fine for now fingers crossed and will update git ignore